swift/apple/FirezoneKit/Sources/FirezoneKit/Keychain/Keychain.swift

```diff
@@ -19,7 +19,6 @@ public enum KeychainError: Error {
 
 public actor Keychain {
   public static let shared = Keychain()
-  private let workQueue = DispatchQueue(label: "FirezoneKeychainWorkQueue")
```

Comment:
Turns out the workQueue is completely unneeded here and avoids a warning about the `[CFString: Any]` type not conforming to `Sendable` in order to be safely captured into the queue closures.